LLama/LLamaInstructExecutor.cs

```diff
@@ -114,7 +114,6 @@ protected override void PreprocessInputs(string text, InferStateArgs args)
             if (_is_prompt_run)
             {
                 // When running the first input (prompt) in inteactive mode, we should specially process it.
-                text = " " + text;
```

Comment:
do we really want to remove this? I had a load of weird issues one time when I didn't add a leading space to inputs.

Reply:
I should learn to read 🤦‍♂️